messaging/propagate.go

```diff
@@ -17,6 +18,9 @@ func init() {
 	network.RegisterMessage(PropagateReply{})
 }
 
+// How long to wait before timing out on waiting for the time-out.
+const initialWait = 100000
```

Comment:
It jokingly describes the bad behaviour of the protocol: instead of correctly waiting for the chosen timeout by the user, which comes through 
```
type PropagationFunc func(el *onet.Roster, msg network.Message, msec int) (int, error)
```
This is in fact a timeout _before_ the chosen time-out is set. Probably would need to be re-designed, so that it doesn't need to have a hard-coded timeout before waiting for the chosen timeout.

So instead of patching this with an exported variable, I'd keep it like this. It's bad and should be re-designed...